docs/spark/tutorials/get-started.md

```diff
@@ -143,48 +155,34 @@ Open *Program.cs* in Visual Studio Code, or any text editor, and replace all of
 ```csharp
 using Microsoft.Spark.Sql;
 
-namespace MySparkApp
+namespace HelloSpark
 {
     class Program
     {
         static void Main(string[] args)
         {
-            // Create a Spark session.
-            var spark = SparkSession
-                .Builder()
-                .AppName("word_count_sample")
-                .GetOrCreate();
-
-            // Create initial DataFrame.
-            DataFrame dataFrame = spark.Read().Text("input.txt");
-
-            // Count words.
-            var words = dataFrame
-                .Select(Functions.Split(Functions.Col("value"), " ").Alias("words"))
-                .Select(Functions.Explode(Functions.Col("words"))
-                .Alias("word"))
-                .GroupBy("word")
-                .Count()
-                .OrderBy(Functions.Col("count").Desc());
-
-            // Show results.
-            words.Show();
-
-            // Stop Spark session.
-            spark.Stop();
+            var spark = SparkSession.Builder().GetOrCreate();
+            DataFrame df = spark.Read().Json("people.json");
+            df.Show();
         }
     }
 }
 ```
 
-### 4. Add data file
+### 4. Create and add a data file
+
+Open your command prompt or terminal and navigate into your app folder.
```

Comment:
```suggestion
Open your command prompt or terminal and navigate into your app folder on **Windows** and **MacOS**.
```

Reply:
Does Ubuntu not have an app folder?

Reply:
Nope, it has. I was thinking the language identifiers for Windows, MacOS(both bash) vs Ubuntu(shell) are different. If it won't affect customers to use it. I am fine with keep only without separating 3 OS systems. Same apply for the questions below.